packages/foreman/rubygem-redfish_client/rubygem-redfish_client.spec

```
@@ -76,11 +46,11 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%exclude %{gem_instdir}/.codeclimate.yml
+%{gem_instdir}/.codeclimate.yml
```

```suggestion
%exclude %{gem_instdir}/.codeclimate.yml
```

https://github.com/theforeman/foreman-packaging/pull/8371